Core/Object Arts/Dolphin/Base/Delay.cls

```diff
@@ -44,6 +44,14 @@ Example usage:
 
 	^self resumptionTime <= anotherDelay resumptionTime!
 
+basicCancel
+	^self == Current
```

Comment:
The ^-return is presumably from an extract method refactoring, and not actually needed here?